### PR TITLE
added incomplete tag

### PR DESCRIPTION
**Pull request by AuraOfTheDawn**
no wiki

### CODE REVIEW

File: games/shovel-knight/game.dat [hunk 1]
```diff
@@ -1,5 +1,6 @@
 title = "Shovel Knight"
 description = "Shovel Knight is a 2D side-scrolling platform game presented in 8-bit graphics in which players control the eponymous protagonist as he collects treasure and fights against the Order of No Quarter."
+incomplete = true
```
Comment (Schplee) [inline games/shovel-knight/game.dat]: The game's graphical glitches don't necessitate 3 screenshots to show off the condition.

File: games/pokemon-mystery-dungeon-gates-to-infinity/game.dat [hunk 2]
```diff
@@ -1,5 +1,6 @@
 title = "Pokémon Mystery Dungeon: Gates to Infinity"
 description = "Pokémon Mystery Dungeon: Gates to Infinity is a role-playing game in the Pokémon franchise developed by Spike Chunsoft and published by Nintendo for the Nintendo 3DS. It is the ninth installment in the Pokémon Mystery Dungeon series."
+incomplete = true
```
Comment (Schplee) [inline games/pokemon-mystery-dungeon-gates-to-infinity/game.dat]: Everything missing besides one screenshot doesn't warrant the `incomplete` field.

File: games/aero-porter/game.dat [hunk 3]
```diff
@@ -1,5 +1,6 @@
 title = "Aero Porter"
 description = "Aero Porter is a Simulation Game developed by Vivarium Inc. and published by Level-5. Aero Porter was originally released at Retail as part of the Guild01 compilation in Japan. This page concerns itself with the separate eShop release. For information on the Japanese Retail release it was a part of, please see the page for Guild01."
+incomplete = true
```
Comment (Schplee) [inline games/aero-porter/game.dat]: Everything missing besides one screenshot doesn't warrant the `incomplete` field.

**Comment by Schplee** [comment]
LGTM

File: games/weapon-shop-de-omasse/game.dat [hunk 4]
```diff
@@ -1,5 +1,6 @@
 title = "WEAPON SHOP de OMASSE"
 description = "Groove along to the beats of the hammer falls, tapping your Nintendo 3DS Touch Screen to the rhythm in order to forge your shop's weapons. The higher your CHAIN, the stronger and sharper the weapon will turn out! Any blacksmith worth his steel also knows that you have to strike while the iron is hot! Make sure to watch the weapon's temperature, which will affect its critical hit rate, durability, and special properties!"
+incomplete = true
```
Comment (TheKoopaKingdom) [inline games/weapon-shop-de-omasse/game.dat]: This game is only missing a screenshot, it can be considered complete.

**Comment by Leo626** [comment]
These are missing screenshots:

- [Castlevania Lords Of Shadow - Mirror of Fate](https://citra-emu.org/game/castlevania-lords-of-shadow-mirror-of-fate/)
- [The Legend of Legacy](https://citra-emu.org/game/legend-of-legacy/)

These are missing hardware information:
- [Mario Kart 7](https://citra-emu.org/game/mario-kart-7/)
- [Story of Seasons: Trio of Towns](https://citra-emu.org/game/story-of-seasons-trio-of-towns/)

**Pull request merged**